### DISCUSSION TITLE
How do you reset the BIT_TOKEN?

### DISCUSSION THREAD

**Discussion by marcusruddickepisource** [teambit/bit#6878] [Help]
There was a [security alert from circleci](https://circleci.com/blog/january-4-2023-security-alert/) and I am currently working on rotating secrets. 

How do you reset the [BIT_TOKEN](https://bit.dev/docs/getting-started/collaborate/setup-ci/) used for bit repositories?

**Comment by ranm8** [comment]
You can reset tokens and use them from CircleCI using Bit Cloud Access Tokens. Learn more on [managing Access Tokens here](https://bit.dev/reference/ci/github-actions#generating-an-access-token).
